Database/connect.sh

```diff
@@ -0,0 +1,3 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
+psql --host $DB_HOST -U $DB_USER -p $DB_PORT $DB_NAME
```

Comment:
```suggestion
psql --host $DB_HOST -U $DB_USER -p $DB_PORT $DB_NAME -f schema.sql
```
Should it be this to connect to the schema?